contributors/devel/pull-requests.md

```diff
@@ -81,7 +81,13 @@ For PRs with a release note:
     Your release note here
     ```
 
-For PRs without a release note:
+For PRs require addtional action for users, include the string "action required" (case insensitive) in the release note:
```

Comment:
"For PRs that require additional action from users switching to the new release" or something like that would be a better start to this. I was previously confused about what "action required" meant, we should make it clear that it is action required of users upgrading to the new release, not action required of the PR author to improve the release note.